fe/fe-core/src/main/java/com/starrocks/connector/ConnectorMetadata.java

```diff
@@ -60,7 +60,7 @@ default Table getTable(String dbName, String tblName) {
         return null;
     }
 
-    default void createDb(CreateDbStmt stmt) throws DdlException {}
+    default void createDb(String clusterName, String dbName) throws  DdlException, CreateExistException {}
```

Comment:
```suggestion
    default void createDb(String dbName) throws  DdlException, CreateExistException {}
```
This is OK.
Use DEFUALT_CLUSTER to be compatible with old version

Reply:
Handle clusterName in LocalMetadata because there is no cluster concept for other data source like Hive.